### DISCUSSION TITLE
Correct ESP32-S3 gpio to connect to 8bit TFT

### DISCUSSION THREAD

**Discussion by mike-rankin** [Bodmer/TFT_eSPI#2614] [New processor/display support or capability request]
I'm trying to connect my ESP32-S3 to an 8-bit BuyDisplay [tft](https://www.buydisplay.com/1-69-inch-color-280x240-round-rectangle-ips-tft-lcd-display-panel-st7789) using Setup206_LilyGo_T_Display_S3.h
The library pins are:
```
#define TFT_DC 7
#define TFT_RST 5
#define TFT_WR 8
#define TFT_RD 9

#define TFT_D0 39
#define TFT_D1 40
#define TFT_D2 41
#define TFT_D3 42
#define TFT_D4 45
#define TFT_D5 46
#define TFT_D6 47
#define TFT_D7 48
```
The display I'm using has
**CS
RS 
WR
RESET**
and D0 -D7

I've connected:
GPIO------------------Display 
TFT_DC 7     to            RS        (RS and Data look like they connect together)
TFT_RST 5    to            RESET  (Both reset)
TFT_WR 8     to           WR       (WR to WR)
TFT_RD 9      to           Not sure what pin to connect TFT_RD to on the display or if can be left not connected to anything

I can connect the display CS to any gpio pin

![Display](https://user-images.githubusercontent.com/4991664/234582647-3af290c7-53f6-42ec-945f-30f7abba9ada.png)

**Comment by Bodmer** [comment]
If the display does not have a RD pin then don't define TFT_RD.
